tests/functional/testServerSearch.js

```diff
@@ -119,7 +119,7 @@ describe("Server list", function () {
     assert.strictEqual(vpn.lastNotification().title, "VPN Connected");
     assert.strictEqual(
       vpn.lastNotification().message,
-      `Connected to ${currentCity}, through ${currentCity}`
+      `Connected through ${currentCity} via ${currentCity}`
```

Comment:
Can you file a Jira issue to have a test for a multi hop notification with 2 different cities?